packit/cli/dist_git_init.py

```diff
@@ -129,6 +133,7 @@
     help="Commit message used when creating a PR (also for the title) or pushing to dist-git. "
     f"Default: {COMMIT_MESSAGE!r}",
 )
+@click.argument("other_args", nargs=-1, type=click.UNPROCESSED)
```

Comment:
I think this can be used as well:

```
    nargs=2,
    type=click.Tuple([str, int]),
    multiple=True,
```

Reply:
this doesn't work as `multiple` is valid only for options and not arguments, if I am checking correctly. Or did you mean it differently?

Reply:
Yes, it can be used with options (which is not an issue here I think). It can simplify the parsing.

We can even implement `key=value` parsing but that's probably an overkill..;)